lib/path-utils.ts

```diff
@@ -14,11 +14,22 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { normalize, relative } from 'path';
+import { FormatInputPathObject, ParsedPath, posix } from 'path';
 
-// Export all of node's path functions, so that users of this
+// Export all of node's POSIX path functions, so that users of this
 // module need only import this file
-export * from 'path';
+export const basename: (p: string, ext?: string) => string = posix.basename;
```

Comment:
These are typed to match the `path` typing right?  In that case it'd be nicer to do
```suggestion
export const basename: typeof path.basename = posix.basename;
```

Reply:
Actually, in this case there is no point in specifying the type at all, as it is inferred anyway:
```
export const basename = posix.basename;
```
I've done just that.